controls/V-72317.rb

```diff
@@ -63,4 +72,16 @@
 finding."
   tag "fix": "Remove all unapproved tunnels from the system, or document them with
 the ISSO."
+
+    @grep_ipsec_conf = inspec.command("grep -i conn /etc/ipsec.conf").stdout.split("\n")
+    @grep_ipsec_d = inspec.command("grep -i conn /etc/ipsec.d/*.conf").stdout.split("\n")
```

Comment:
This needs to be changed to a full processor that will follow the `include` references in the main `ipsec.conf` file.

You can include things from literally anywhere and we're going to miss a lot of we don't follow the links.

A small modification of this will probably work just fine https://gist.github.com/trevor-vaughan/df6a3a0ca42a2c6f59b51d7375fe151a

@jburns12 If you want to move this to an additional ticket, we can push this through as it stands. I just don't want to lose the fact that it's not actually properly processing everything.

Reply:
@trevor-vaughan That makes sense to me...I'll open another ticket in the next couple days. Since my time doing this work is limited at this point, this might end up getting picked up by another MITRE team member.